Tests/test_GenBank.py

```diff
@@ -1519,9 +1519,7 @@ def test_record_parser_13(self):
             record = next(records)
             length = 2007
             locus = "AB000048"
-            definition = (
-                "Feline panleukopenia virus DNA for nonstructural protein 1, complete cds"
-            )
+            definition = "Feline panleukopenia virus DNA for nonstructural protein 1, complete cds"
```

Comment:
Isn't this strange? This doesn't fit in one line, does it? See the similar but shorter string in line 1600/1596, which is put on an extra line.

Reply:
This gives a line of 99, the old version had three lines the longest of which was 90. So both break black's target of 88, not much to choose between them (and different versions of black differed on this). I prefer the one line version.

We ought to break the string in two, I didn't notice in review,

Reply:
I'm not sure why black has removed the parentheses here and not on 1596.
usually if there is few PEP8 (line too long), I'll break them but this file have many so I have left that, I think that should be part of other Issue (PEP Compliance)?.

Reply:
I was just curious about the 'different' behavior of black, but didn't realize that in this case, both versions produce too long lines. The PR is already merged, so I wouldn't re-open it.

Reply:
Agreed - there is still an overly long line (due to the unbroken string), but there are a *lot* of overly long lines in the test suite. Something for another day.